apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/network_constraints.py

```diff
@@ -114,8 +127,17 @@ def parse_cidr_list(cidrs):
                             {'cidr': cidrs, 'exc': e})
                 return None
 
+        def parse_allow_vm_names(vm_names):
+            try:
+                return [name.strip() for name in vm_names.split(',')]
```

Comment:
ok. Looks like I can use re.compile(<regex>) to see if its valid or not. Will add that then.